internal/stat_unix.go

```diff
@@ -1,5 +1,5 @@
-//go:build !windows
-// +build !windows
+//go:build !windows && !plan9
+// +build !windows,!plan9
```

Comment:
```suggestion
```
Our `go.mod` minimum supported version is now set to Go 1.21, and [Go 1.18 ends support for these](https://go.dev/doc/go1.18#go-build-lines), suggesting the lines be removed because [they're error prone](https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md).